lib/perfBaseTest.py

```diff
@@ -203,8 +203,8 @@ def tearDown(self):
         # output result
         if self.sikuli_status == 0:
             if hasattr(self, "crop_data"):
-                resultHelper.result_calculation(self.env, self.exec_timestamp_list, self.crop_data, int(os.getenv("CALC_SI")), int(os.getenv("ENABLE_WAVEFORM")))
+                resultHelper.result_calculation(self.env, self.exec_timestamp_list, self.crop_data, int(os.getenv("CALC_SI")), int(os.getenv("ENABLE_WAVEFORM"), os.getenv("PERFHERDER_REVISION")))
             else:
-                resultHelper.result_calculation(self.env, self.exec_timestamp_list, calc_si=int(os.getenv("CALC_SI")), waveform=int(os.getenv("ENABLE_WAVEFORM")))
+                resultHelper.result_calculation(self.env, self.exec_timestamp_list, calc_si=int(os.getenv("CALC_SI")), waveform=int(os.getenv("ENABLE_WAVEFORM")), revision=os.getenv("PERFHERDER_REVISION"))
```

Comment:
If there is no `os.getenv("PERFHERDER_REVISION")`, it will return `None`.
And https://github.com/Mozilla-TWQA/Hasal/pull/321/files#diff-1ad9d9500dccab2cb82c1df2978bb934R105 will dump it from `None` to `null`, not `""` (empty string).